src/auth/auth.controller.ts

```diff
@@ -1,25 +1,36 @@
 import type { Request, Response } from "express-serve-static-core";
 import { StatusCodes } from "http-status-codes";
+import { AuthService } from "./auth.service";
+
+const authService = new AuthService();
 
 export default class AuthController {
-  async register(req: Request, res: Response): Promise<void> {
-    res.status(StatusCodes.OK).json({ message: "It works!" });
+  async register(req: Request, res: Response) {
```

Comment:
I propose to use arrow function everywhere where it is possible to make code consistent throughout the app.